### PR TITLE
[JEN-981] Add cosmic support

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Review by abychko — COMMENTED** [review]
that's OK but jobs will not work without appropriate image in jenkins

File: docker/install-deps [hunk 1]
```diff
@@ -96,6 +96,11 @@ if [ -f /usr/bin/apt-get ]; then
         echo "waiting"
         sleep 1
     done
+    if [[ $(lsb_release -sc)  = cosmic ]]; then
+        apt-get -y install libssl1.0-dev libcurl4-gnutls-dev
+    else
+        apt-get -y install libssl-dev
```
Comment (laurynas-biveinis) [inline docker/install-deps:102]: why libssl-dev needs installing for not-cosmic?

Comment (EvgeniyPatlan) [reply docker/install-deps:102]: @laurynas-biveinis there was an issues during build for newest libssl and that's why for cosmic we install libssl1.0 which is the same as libssl on all other systems

Comment (laurynas-biveinis) [reply docker/install-deps:102]: Please escalate such issues: openssl is different for end users too, depending on the version. libssl-dev is 1.1.1 on Cosmic, which is very different from 1.1.0, which we do support. Does cosmic package 1.1.0 too?

Comment (EvgeniyPatlan) [reply docker/install-deps:102]: @laurynas-biveinis  here is the package info from xenial and cosmic with such setup:
xenial:

ubuntu@ubuntu-xenial:~$ dpkg -l | grep libssl
ii  libssl-dev:amd64                 1.0.2g-1ubuntu4.14                 amd64        Secure Sockets Layer toolkit - development files
ii  libssl-doc                       1.0.2g-1ubuntu4.14                 all          Secure Sockets Layer toolkit - development documentation
ii  libssl1.0.0:amd64                1.0.2g-1ubuntu4.14                 amd64        Secure Sockets Layer toolkit - shared libraries

cosmic:

vagrant@ubuntu1810:~$ dpkg -l | grep libssl
ii  libssl1.0-dev:amd64                   1.0.2n-1ubuntu6.1                 amd64        Secure Sockets Layer toolkit - development files
ii  libssl1.0.0:amd64                     1.0.2n-1ubuntu6.1                 amd64        Secure Sockets Layer toolkit - shared libraries
ii  libssl1.1:amd64                       1.1.1-1ubuntu2                    amd64        Secure Sockets Layer toolkit - shared libraries


So as you can see the versions in such configurations are the same

Comment (laurynas-biveinis) [reply docker/install-deps:102]: Why xenial is relevant here?

Anyway, the patch LGTM, please prepare 5.6 and 5.7 versions

**Review by laurynas-biveinis — APPROVED** [review]

**Review by abychko — APPROVED** [review]

**Comment by laurynas-biveinis** [comment]
Why Cosmic does not appear in the job matrix even this PR added this? https://ps.cd.percona.com/view/8.0/job/percona-server-8.0-param/195/

**Comment by EvgeniyPatlan** [comment]
@laurynas-biveinis fixed. It should be changed in jenkins config not in PR

**Pull request merged**